.github/workflows/ci.yml

```diff
@@ -3,7 +3,9 @@ on:
   pull_request:
     branches: [master]
   push:
-    branches: [master]
+    branches:
+      - master
+      - 'maintainers/**'
```

Comment:
Since only maintainers can create branches, why not just enable that on all branches? :thinking:

Reply:
Good question @leogr! this is because if you open a pull request from an upstream branch GH jobs will run twice since you have both a pull-request and an upstream branch. The idea here is to use  `maintainers/` branches as test and then open the pull request from another branch called in a different way, in this way jobs will run just one time

Reply:
No way to tell the GHA system to dedup jobs with the same commit id? :thinking: 

I'm not contrary to this solution, anyway there're two things I don't like:
- This introduces a convention that maintainers should know, and it's easy to forget
- Occasionally, a testing branch may become PR; in that case, if one by mistake reuses the same branch, the jobs will still run twice

In other repositories, we sometimes open dummy PRs that will never be merged (i.e, `DO NOT MERGE THIS PR`) for the same purpose (just running the CI jobs). 
IMO, it's an alternative to consider since that wouldn't require an explicit CI config, makes the practice of manually testing some code more transparent, and can be used by non-maintainers too.

Just my 2 cents.

what do other maintainers think? cc @falcosecurity/falco-maintainers